src/controller/tests/data_model/TestRead.cpp

```diff
@@ -1724,7 +1724,8 @@ void TestReadInteraction::TestReadHandler_MultipleSubscriptions(nlTestSuite * ap
         NL_TEST_ASSERT(apSuite,
                        Controller::SubscribeAttribute<TestCluster::Attributes::ListStructOctetString::TypeInfo>(
                            &ctx.GetExchangeManager(), sessionHandle, kTestEndpointId, onSuccessCb, onFailureCb, 0, 20,
-                           onSubscriptionEstablishedCb, nullptr, false, true) == CHIP_NO_ERROR);
+                           onSubscriptionEstablishedCb, nullptr, false, true, NullOptional,
+                           System::Clock::Milliseconds32(20000)) == CHIP_NO_ERROR);
```

Comment:
Please document why this needs a non-default timeout.

Reply:
The old MRP idle timeout was hiding cases where long interaction timeouts were required.  Because 4 send attempts * (5 seconds + margin) between each send attempt, the MRP idle interval would hide cases where IM timeout should be set to >20 seconds.  Do you want this documented as a comment in the test or just here in the PR?

The need for extended IM timeout for network scan that exceeds the duration of the scan is fairly clear.
For the multiple subscription test, the assumption is that processing of subscriptions is serialized and therefore one of the subscription responses will come back much later than the others.

Reply:
For this case, won't the server synchronously process requests and return a response+piggybacked ack for each?  If so, shouldn't the default IM timeout derived from MRP timeout still be sufficient?

Reply:
For network scan, don't we send a forced standalone ack on the server, precisely because it's a long-running operation?

And for subscriptions, I would expect responses to any given message to happen fairly quickly.  Subscription processing is not serialized.